### PR TITLE
chore: Rename `master` branch to `main`

### PR DESCRIPTION
**Pull request by pgrzesik**
Rename `master` to `main` branch

### CODE REVIEW

**Comment by pgrzesik** [comment]
Tests will run only after we actually change the `master` to `main`, that's why I'm asking for a review beforehand @medikoo

File: lib/plugins/plugin/lib/utils.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ const { log, writeText, style } = require('@serverless/utils/log');
 
 module.exports = {
   async getPlugins() {
-    const endpoint = 'https://raw.githubusercontent.com/serverless/plugins/master/plugins.json';
```
Comment (medikoo) [inline lib/plugins/plugin/lib/utils.js:12]: This I think needs to stay intact

File: test/unit/lib/plugins/plugin/lib/utils.test.js [hunk 2]
```diff
@@ -56,7 +56,7 @@ describe('PluginUtils', () => {
     });
 
     it('should fetch and return the plugins from the plugins repository', () => {
-      const endpoint = 'https://raw.githubusercontent.com/serverless/plugins/master/plugins.json';
```
Comment (medikoo) [inline test/unit/lib/plugins/plugin/lib/utils.test.js:59]: Same here

**Comment by pgrzesik** [comment]
good catch @medikoo - fixed

**Review by medikoo — APPROVED** [review]
Looks great 👍

**Pull request merged**